examples/jwt.js

```diff
@@ -31,10 +31,14 @@ async function main() {
     key: keys.private_key,
     scopes: ['https://www.googleapis.com/auth/cloud-platform']
   });
-  await client.authorize();
   const url = `https://www.googleapis.com/dns/v1/projects/${keys.project_id}`;
   const res = await client.request({ url });
   console.log(res.data);
+
+  // After acquiring an access_token, you may want to check on the audience, expiration,
+  // or original scopes requested.  You can do that with the `getTokenInfo` method.
+  const info = await client.getTokenInfo(client.credentials.access_token);
```

Comment:
Nit: Use `tokenInfo` to match the oauth2.js and README example.

Reply:
fixed